pydocx/export/html.py

```diff
@@ -361,6 +361,114 @@ def export_paragraph_property_indentation(self, paragraph, results):
+                return prev_level_paragraphs[-1]
+
+            if prev_level_id == 0 and not prev_level_paragraphs:
+                # this is an edge case with older version of word when it may contain a sublist
+                # into a separate num_id.
+                break
+
+            level_id -= 1
+
+        return None
+
+    def export_listing_paragraph_property_indentation(self, paragraph, level_properties,
+                                                      include_text_indent=False):
+        style = {}
+
+        if not level_properties or not paragraph.has_numbering_properties:
+            return style
+
+        default_level_indentation = paragraph.get_numbering_default_level_indentation()
+
+        paragraph_properties = paragraph.properties
+
+        level_id = int(paragraph_properties.numbering_properties.level_id)
+        num_id = paragraph_properties.numbering_properties.num_id
+
+        level_ind_left = level_properties.to_int('indentation_left', default=0)
+        level_ind_hanging = level_properties.to_int('indentation_hanging', default=0)
+
+        paragraph_ind_left = paragraph_properties.to_int('indentation_left', default=0)
+        paragraph_ind_hanging = paragraph_properties.to_int('indentation_hanging', default=0)
+        paragraph_ind_first_line = paragraph_properties.to_int('indentation_first_line',
+                                                               default=0)
+
+        left = paragraph_ind_left or level_ind_left
+        hanging = paragraph_ind_hanging or level_ind_hanging
+
+        # at this point we have no info about indentation, so we keep the default one
+        if not left and not hanging:
+            return style
+
+        if num_id not in self.numbering_level_listing_track:
+            # by default there are only 9 numbering levels in docx(0 indexed)
+            self.numbering_level_listing_track[num_id] = [[] for _ in range(10)]
+        if paragraph not in self.numbering_level_listing_track[num_id][level_id]:
+            self.numbering_level_listing_track[num_id][level_id].append(paragraph)
+
+        # by default left contains hanging as well, so we remove it
+        left -= hanging
+
+        if level_id == 0:
+            # because html ul/ol/li elements have there default indentations
+            # we remove the default word one as well
+            # this way we will have as near as possible migration to html
+            left -= (default_level_indentation['left'] - level_ind_hanging)
+
+            # first line are added left margins
+            if paragraph_ind_first_line:
+                left += paragraph_ind_first_line
+
+        if level_id > 0:
+            # for nested levels we need to add indentation based on parent level
+            prev_paragraph = self.get_previous_level_paragraph(num_id, level_id)
+            if prev_paragraph:
+                prev_left_level_indentation = prev_paragraph.get_numbering_level().\
```

Comment:
We go with the style of opening the parens and then indenting the next line, rather than using `\`.